packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -409,12 +409,15 @@ const schema: SchemaType<"Posts"> = {
     order: 10,
     group: formGroups.adminOptions,
     onCreate: ({document: post}) => {
-      if(!post.sticky) {
+      if(!isEAForum && !post.sticky) {
```

Comment:
This doesn't seem like it ought to be doing anything given the `defaultValue` set above; might be wrong though